src/script/interpreter.cpp

```diff
@@ -60,6 +60,38 @@ static inline void popstack(std::vector<valtype>& stack)
     stack.pop_back();
 }
 
+static inline void pushasset(std::vector<valtype>& stack, const CConfidentialAsset& asset)
+{
```

Comment:
Can you `assert` here that `!asset.IsNull()`? (I believe this is enforced for any transaction input...although I wonder what happens for e.g. coinbase transactions?)

Reply:
Or you could just push `0` onto the stack in case the asset is null, and not worry about whether this is actually possible.